### PR TITLE
OBS-1387 Add administrator users; configure ssh and sudo

### PR DESCRIPTION
**Pull request by mdpearson**
@jmiranda This PR creates new users, `jmiranda` and `soldevelo`, with ssh and sudo access. Artur had requested this; I'll file two separate tickets and assign them to each of you with next steps.

### CODE REVIEW

File: ansible/install_dependencies.yml [hunk 1]
```diff
@@ -77,46 +77,120 @@
...
+          system: true
+          uid: 511
+        - group: staff
+          groups: tomcat
+          name: bamboo
+          system: false
+          uid: 1509
+        - group: staff
+          groups: tomcat
+          name: jmiranda
```
Comment (jmiranda) [inline ansible/install_dependencies.yml:112]: Are users something (like server inventory) that can/should be pulled out into a separate configuration.

File: ansible/install_dependencies.yml [hunk 2]
```diff
@@ -77,46 +77,120 @@
...
+      loop:
+        - key_file: public_keys/anadolny_ed25519.pub
+          user: anadolny
+        - key_file: public_keys/awalkowiak_ed25519.pub
+          user: awalkowiak
+        - key_file: public_keys/bamboo_ed25519.pub
+          user: bamboo
+        - key_file: public_keys/drodzewicz_ed25519.pub
+          user: drodzewicz
+        - key_file: public_keys/jmiranda_ed25519.pub
```
Comment (jmiranda) [inline ansible/install_dependencies.yml:161]: Same as with users, is there a way to pull this out to be more generic. I'm thinking the same thing needs to happen with inventory. We almost need the devops repo to be the place where we define what needs to happen and then we have other repos which provide specifics about servers, users, etc. For example, we want the generic devops to do everything and then a specific "openboxes", "pih", "sbhf" repos to tell us where and who to do those things to.

**Review by jmiranda — APPROVED** [review]
LGTM; 

I've added some general thoughts on where we might want to go in the future with making our ansible scripts more generic.

File: ansible/install_dependencies.yml (hunk 2 above)
Comment (jmiranda) [reply ansible/install_dependencies.yml:161]: I know this isn't the right way to do it but I'm imagining something like this
```
mkdir pih-devops
cd pih-devops
git clone github.com/openboxes/openboxes-devops
git clone github.com/openboxes/openboxes-pih-devops
```
In OOP, the analogy would be like openboxes-devops is the abstract base class and openboxes-pih-devops would implementation. 

You could even think about making a template repo that can be cloned and imported into a separate organization
```
git clone github.com/openboxes/openboxes-devops
git clone github.com/partnersinhealth/partnersinhealth-devops
```

File: ansible/install_dependencies.yml (hunk 1 above)
Comment (jmiranda) [reply ansible/install_dependencies.yml:112]: Here's an example as it pertains to defining users within their own files.
https://stackoverflow.com/questions/57314773/adding-multiple-users-to-a-linux-system-with-ansible

This is how hugo (CMS used by openboxes.com) does content management and I kinda like it. Same idea can be applied to server inventory.

File: ansible/install_dependencies.yml (hunk 2 above)
Comment (mdpearson) [reply ansible/install_dependencies.yml:161]: I agree. This is one of those things that is growing organically and should be refactored. I'm going to pull your PR comments to a new OBS ticket (OBS-1414)

Comment (mdpearson) [reply ansible/install_dependencies.yml:161]: but for the time being I also want people to be able to log into these hosts. I'm going to merge

**Pull request merged**